reactive/kubernetes_worker.py

```diff
@@ -511,6 +519,7 @@ def start_worker():
     restart_unit_services()
     update_kubelet_status()
     set_state('kubernetes-worker.label-config-required')
+    set_state('nrpe-external-master.reconfigure')
```

Comment:
we don't know if we have `nrpe` here.  guard this with a `is_flag(available) && set_state(reconfigure)`?

Reply:
i kinda want to punch your dog's face.  if nrpe isn't available, let's not reconfigure it.

Reply:
ah, never mind this.  I misread the upgrade-charm handler above as using `if nrpe.reconfigure, update_nrpe_config()`.  In fact, it says `if nrpe.AVAILABLE, update_nrpe_config()`, so that's good.  I was afraid we would always go into `update_nrpe_config` without nrpe being present.  My bad, this is i guess ok.